Makefile

```diff
@@ -25,8 +23,8 @@ usage:
 	@echo "  format_check        -- Check source code formatting without making changes"
 	@echo "  cabalfmt            -- Apply cabal formatting with cabal-fmt"
 	@echo "  cabalfmt_check      -- Check cabal files for formatting errors without making changes"
-	@echo "  nixfmt              -- Apply nix formatting with nixfmt"
-	@echo "  nixfmt_check        -- Check nix files for format errors"
+	@echo "  nixpkgsfmt          -- Apply nix formatting with nixpkgs-fmt"
```

Comment:
What's with the preference of nixpkgsfmt over nixfmt?

Reply:
nixpkgsfmt is the one used by nixpkgs, so more standard basically AFAIK